Jenkinsfile

```diff
@@ -120,6 +115,13 @@ pipeline {
       junit healthScaleFactor: 200.0,           \
         testResults: '*/build/reports/*.xml'
 
+      timeout (1) {
```

Comment:
Is this a one second timeout? Is that enough in every situation?

Reply:
It's one minute, which ought to be enough.